cmake/modules/FindBabbleSim.cmake

```diff
@@ -60,3 +60,34 @@ endif()
 #Many apps cmake files (in and out of tree) expect these environment variables. Lets provide them:
 set(ENV{BSIM_COMPONENTS_PATH} ${BSIM_COMPONENTS_PATH})
 set(ENV{BSIM_OUT_PATH} ${BSIM_OUT_PATH})
+
+# Let's check that it is new enough and built,
+# so we provide better information to users than a compile error:
+
+function(bsim_handle_not_built_error)
```

Comment:
this and the other function uses CMake variables defined outside of this function.

Probably make a comment that those functions should only be used internally in this CMake module, because calling them outside may have unexpected behavior.

Reply:
commit updated as requested